### PR TITLE
fix isImport externalcluster

### LINKED ISSUE #8462: Fix method to check if EKS was imported
**User Story**
Fix method to check if EKS was imported 

**Acceptance criteria**

### PR DESCRIPTION
**Pull request by imharshita**
Signed-off-by: Harshita <imharshita@github.com>

**What this PR does / why we need it**: fix isImport externalcluster gke, aks. eks

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #8462, #8463, #8453 

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: pkg/crd/kubermatic/v1/preset.go [hunk 1]
```diff
@@ -346,9 +346,9 @@ func (s EKS) IsValid() bool {
 type AKS struct {
 	ProviderPreset `json:",inline"`
 
-	TenantID       string `json:"tenantId"`
-	SubscriptionID string `json:"subscriptionId"`
-	ClientID       string `json:"clientId"`
+	TenantID       string `json:"tenantID"`
```
Comment (zreigz) [inline pkg/crd/kubermatic/v1/preset.go]: @maciaszczykm if we change it we have to update presets in the UI. We can also leave like it is right now, WDYT?

Comment (maciaszczykm) [reply pkg/crd/kubermatic/v1/preset.go]: I'd keep it as it is to be honest. It's the same for Azure I think.

Comment (zreigz) [reply pkg/crd/kubermatic/v1/preset.go]: OK, so @imharshita please revert those changes for the `preset.go`

**Pull request merged**